<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ramses</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icon-large</normaloff>:/icons/icon-large</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="mainLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QVBoxLayout" name="duqf_settingsLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="titleWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>13</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="duqf_reinitSettingsButton">
             <property name="toolTip">
              <string>Reset all settings to their default values and restart the application</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="duqf_resources.qrc">
               <normaloff>:/icons/reinit</normaloff>:/icons/reinit</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="duqf_closeSettingsButton">
             <property name="toolTip">
              <string>Close settings</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="duqf_resources.qrc">
               <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLogIn">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/login</normaloff>:/icons/login</iconset>
   </property>
   <property name="text">
    <string>Log in...</string>
   </property>
  </action>
  <action name="actionLogOut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/logout</normaloff>:/icons/logout</iconset>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionUserProfile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/user-settings</normaloff>:/icons/user-settings</iconset>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/admin-settings</normaloff>:/icons/admin-settings</iconset>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </action>
  <action name="actionProjectSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/project-settings</normaloff>:/icons/project-settings</iconset>
   </property>
   <property name="text">
    <string>Project Settings</string>
   </property>
  </action>
  <action name="actionUserFolder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/reveal-folder</normaloff>:/icons/reveal-folder</iconset>
   </property>
   <property name="text">
    <string>User folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="duqf_resources.qrc"/>
 </resources>
 <connections/>
</ui>
